internal/app/api/user/user_suite_test.go

```diff
@@ -0,0 +1,236 @@
+package user_test
```

Comment:
should this file be with the user.go file in models?

Reply:
Negative, the models directory will just contain the structs that define shared types and database tables.